script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -657,7 +657,7 @@ Vlov's complexion and anguish have become even more ghoulish.
 -- Page 506, Offset 9950.
 -- あの灼熱の中で<ひ|・><と|・><り|・><だ|・><け|・><世|・><界|・><が|・><違|・><う|・>。
 // あの灼熱の中で一人だけ世界が違う
-Within that scorching heat, he exists in his own separate world.
+Within that scorching heat, he exists in %{i}his own separate world%{/i}.
```

Comment:
Not sure I like this one

Reply:
I think "separate world" being italicized would be fine.